pkg/utils/path.go

```diff
@@ -110,3 +110,13 @@ func FindFilesBySuffix(basePath string, suffixes []string) (map[string][]*string
 
 	return retMap, nil
 }
+
+// FindFilesBySuffixInCurrentDir finds all the immediate files within a given directory that have the specified suffixes
+// Returns an array for string pointers as a list of files
+func FindFilesBySuffixInCurrentDir(basePath string, suffixes []string) ([]*string, error) {
```

Comment:
Are you searching for files in current dir or in `basePath`? 

May be a better name would be `FindFilesBySuffixInCurrentDir `

Reply:
Isn't that the name?

Reply:
Sorry copy paste mistake.
Can we use the `FindFilesBySuffix` method here?

Reply:
No. we don't want a recursive search here. We just want to look up files on a single depth level. No looking into subdirectories.

Reply:
Updating the function name though